src/server/svr_func.c

```diff
@@ -798,7 +798,8 @@ set_rpp_highwater(attribute *pattr, void *pobj, int actmode)
  *		scheduler.   Done here because also need to invalidate the server
  *		state attribute cache.
  *
- * @param[in]	s	-	internal socket used to communicate with the scheduler
+ * @param[in] s		-	internal socket used to communicate with the scheduler
```

Comment:
Is the indentation off?  It shows off on GitHub

Reply:
It looks like it is GitHub issue. The indentation seems fine in my editor.